### PR TITLE
Updating GCDAsyncSocket with new version

### PR DESCRIPTION
**Pull request by dersim-davaod**
This change is [Reviewable](https://reviewable.io/reviews/thaliproject/thali_cordovaplugin/1702)

### CODE REVIEW

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 3. task. (c6641e3)

File: thali/install/setUpTests.sh [hunk 1]
```diff
@@ -37,7 +37,7 @@ prepare_project()
   npm install --no-optional
   node generateServerAddress.js $IPADDRESS
   cd $REPO_ROOT_DIR/..
-  cordova create $TEST_PROJECT_NAME com.test.thalitest $TEST_PROJECT_NAME
+  cordova create $TEST_PROJECT_NAME com.test.thalitet $TEST_PROJECT_NAME
```
Comment (larryonoff) [inline thali/install/setUpTests.sh]: this's wrong change. Please revert it.

File: thali/install/setUpTests.sh [hunk 2]
```diff
@@ -199,10 +199,10 @@ build_ios_if_possible()
 # Please note that functions order is important
 IPADDRESS=${2:-}
 prepare_project ${IPADDRESS}
-add_android_platform
+#add_android_platform
```
Comment (larryonoff) [inline thali/install/setUpTests.sh]: please revert this.

File: thali/install/setUpTests.sh [hunk 3]
```diff
@@ -199,10 +199,10 @@ build_ios_if_possible()
...
 add_ios_platform_if_possible
 install_thali $1 ${IPADDRESS}
-build_android
+#build_android
```
Comment (larryonoff) [inline thali/install/setUpTests.sh]: please revert this.

**Comment by ThaliTester** [comment]
Test 99559613 (c6641e3) build started.

**Comment by ThaliTester** [comment]
Test (Fail) 99559613 build is completed (c6641e3)

See https://github.com/ThaliTester/TestResults/tree/99559613c6641e3_Updating_GCDAsyncSocket_with_new_version_dersim-davaod/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (d63b0ac)

**Comment by ThaliTester** [comment]
Test (Fail) 99559613 build is completed (d63b0ac)

See https://github.com/ThaliTester/TestResults/tree/99559613d63b0ac_Updating_GCDAsyncSocket_with_new_version_dersim-davaod/ for the logs

**Pull request merged**